presentation/src/main/java/com/android254/presentation/speakers/view/SpeakersScreen.kt

```diff
@@ -76,14 +87,7 @@ private fun SpeakersScreen(
     Scaffold(
         topBar = {
             SmallTopAppBar(
-                title = {
-                    Text(
-                        text = stringResource(id = R.string.speakers_label),
-                        fontSize = 24.sp,
-                        fontFamily = FontFamily(Font(R.font.montserrat_regular)),
```

Comment:
`ChaiDCKE22Theme` provides for dark and light color scheme. You can just add an entry to ChaiColors and use it in this text instead of checking `isSystemInDarkTheme` and use it this way `MaterialTheme.chaiColorsPalette.your_color`

Reply:
On it !!